src/utils/process.rs

```diff
@@ -726,6 +726,9 @@ pub mod tests {
         let args = make_string_vec(&["git", "-c", "a=b", "blame", "main.rs"]);
         assert_eq!(guess_git_blame_filename(&args), Args("main.rs".into()));
 
+        let args = make_string_vec(&["git", "blame", "-w", "-C", "main.rs"]);
+        assert_eq!(guess_git_blame_filename(&args), Args("main.rs".into()));
+
```

Comment:
That unfortunately doesn't really fix it: Now `-w` takes an argument (when it does not), which means it "eats" the following `-C`. The problem is `-C[<num>]`, which only takes a non-whitespace argument (or none) -- but this not really modeled at the moment.

Removing `-C` from the previous list should work for most cases. In the future a fallback to take the last argument containing a `.` should be added. And make it use the `CallingProcess` enum as mentioned in the TODO there.

Reply:
I've been tricked by only looking at the synopsis part of the git blame man page when adding `-w` to this list.

I'll look at addressing this TODO and making a variant in `CallingProcess`.